docs/src/pages/components/alert/AlertPropControls.tsx

```diff
@@ -36,9 +35,8 @@ export const AlertPropControls: AlertPropControlsInterface = ({
   body,
   setBody,
 }) => {
-  const { tokens } = useTheme();
   return (
-    <>
+    <Flex direction="column">
```

Comment:
Removed `.amplify-field { margin-bottom }` style, so just remember to wrap forms in a Flex.

Reply:
https://media.giphy.com/media/3WCNY2RhcmnwGbKbCi/giphy.gif